Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - && \
 	apt-get install -y \
 	git \
 	curl \
+	expect \
```

Comment:
add a comment about why it's needed? And also maybe wait until it's actually needed.